app/src/androidTest/java/TestPlan.kt

```diff
@@ -1,20 +1,65 @@
-import androidx.test.rule.ActivityTestRule
-import com.maubis.scarlet.base.MainActivity
-import org.junit.Rule
+import EspressoBaseTest.checkElementIsDisplayed
+import EspressoBaseTest.checkMatchesStringInElement
+import EspressoBaseTest.checkStringInComponentHostIsDisplayed
+import EspressoBaseTest.clickBottomToolbarButton
+import EspressoBaseTest.clickButton
 import org.junit.Test
+import ui.*
 
-class TestPlan {
+class TestPlan : BaseClass() {
 
-    @Rule
-    @JvmField
-    var activityTestRule = ActivityTestRule(MainActivity::class.java)
+    @Test
+    fun checkEmptyNotes() {
+        HomePage().recyclerView.clickRecyclerViewItem(1)
```

Comment:
You are always creating new instance  of `*Page`. It affects efficiency. Maybe it is good idea to refactor it somehow?